Cabal/tests/PackageTests/Tests.hs

```diff
@@ -120,10 +120,6 @@ tests config = do
   -- with the same name and LATER version
   tc "BuildDeps/InternalLibrary3" $ internal_lib_test "internal"
 
-  -- Test that an explicit dependency constraint which doesn't
```

Comment:
Those packages are broken wrt what was written anyways, and there's the functionality to ninja the cabal files on hackage releases anyways.